MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -8084,5 +8091,117 @@ private void gettime()
 
             Module.Memory.SetArray(timePointer, dosTimeStruct.Data);
         }
+
+        /// <summary>
+        ///     Full Screen Editor
+        ///
+        ///     We re-use the FSD, using a pre-defined custom template, field spec, and formats (in Assets Folder)
+        /// </summary>
+        private void bgnedt()
+        {
+            var textLength = GetParameter(0);
+            var textPointer = GetParameterPointer(1);
+            var topicLength = GetParameter(3);
+            var topicPointer = GetParameterPointer(4);
+            var onDoneEditing = GetParameterPointer(6);
+            var flags = GetParameter(8);
+
+            //This is where we'll build the Answers String to pass into the FSD Template
+            var answersString = new StringBuilder(topicLength + textLength);
+
+            //Compile a Topic Answer String
+            answersString.Append($"TOPIC={Encoding.ASCII.GetString(Module.Memory.GetString(topicPointer))}");
+            
+            //Take the input text and split it out on new lines to we can hydrate the individual lines on the template
+            var rawText = Encoding.ASCII.GetString(Module.Memory.GetString(textPointer, true));
+            var split = rawText.Split('\r');
+            var lineNumber = 0;
+            for (var i = 0; i < split.Length; i++)
+            {
+                if (split[i].Length < 79)
+                {
+                    answersString.Append($"LINE{i}={split[i]}\0");
```

Comment:
should be `" LINE"` (with space) like below, or no space in the string here?

Reply:
No spaces in the string. The Answers are null terminated strings and the answer string itself it double null terminated.

Reply:
line 8131 has a space still